webapp/src/ts/modals/mm-modal/mm-modal.component.html

```diff
@@ -1,7 +1,7 @@
 <div role="document" id="{{id}}">
   <div class="modal-header">
     <button type="button" class="cancel close" [disabled]="status?.processing" (click)="onCancel.emit()">&times;</button>
-    <h2 translate>{{titleKey}}</h2>
```

Comment:
For some reason, when the titleKey changes dynamically, the directive doesn't update (and shows the untranslated key), while the pipe does.